guide/improving-dev-environment/package-json-scripts.md

```diff
@@ -0,0 +1,84 @@
+# Setting up package.json scripts
+
+An easy way to run scripts like the start script, the lint script or whatever scripts you use, is by storing them in your `package.json`. After you stored these scripts in your `package.json`, you can just type `npm run start` or `npm run lint` to start your bot or lint your code!
+
+## Getting started
+
+<p class="tip">Before getting started, you'll need to have a `package.json`. If you don't have a `package.json` yet, you can run `npm init -y` to generate one.</p>
+
+If you haven't touched your `package.json` yet (excluding installing dependencies), your `package.json` should look similar to the following:
+
+```json
+{
+  "name": "Making-Bots-With-Discord.js",
+  "version": "1.0.0",
+  "description": "The official guide for [discord.js](https://github.com/discordjs/discord.js), created and maintained by core members of its community.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/devdutchy/Making-Bots-with-Discord.js.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/devdutchy/Making-Bots-with-Discord.js/issues"
+  },
+  "homepage": "https://github.com/devdutchy/Making-Bots-with-Discord.js#readme"
+}
+```
+
+Let's zoom in more. Below `main`, you'll see `scripts`. You can specify your scripts there. In this guide, we'll show how to start and lint your bot using a `package.json` script.
+
+## Adding your first script
+
+<p class="tip">We'll assume you have finished the [Creating your first bot](https://discordjs.guide/#/creating-your-bot/) section of the guide. If you haven't, ensure to follow it first!</p>
```

Comment:
- "... have finished the [Creating your first bot](https://discordjs.guide/#/creating-your-bot/) section" -> "... have finished [creating your first bot](/creating-your-bot)`
- Make sure to use the relative paths instead of hard-coded links when linking to pages within the guide (i.e. `/creating-your-bot/` instead of `https://discordjs.guide/#/creating-your-bot/`